website/docs/guides/version-3-upgrade.html.md

```diff
@@ -121,6 +122,65 @@ output "lambda_result" {
 }
 ```
 
+## Resource: aws_autoscaling_group
+
+### Drift detection enabled for `load_balancers` and `target_group_arns` arguments
+
+If you previously set one of these arguments to an empty list to enable drift detection (e.g. when migrating an ASG from ELB to ALB), this can be updated as follows.
+
+For example, given this previous configuration:
+
+```hcl
+resource "aws_autoscaling_group" "example" {
+  # ... other configuration ...
+  load_balancers = []
+  target_group_arns = [aws_lb_target_group.example.arn]
+}
+```
+
+An updated configuration:
+
+```hcl
+resource "aws_autoscaling_group" "example" {
+  # ... other configuration ...
+  target_group_arns = [aws_lb_target_group.example.arn]
+}
+```
+
+If `aws_autoscaling_attachment` resources reference your ASG configurations, you will need to add the [`lifecycle` configuration block](/docs/configuration/resources.html#lifecycle-lifecycle-customizations) with an `ignore_changes` argument to prevent Terraform non-empty plans (i.e. forcing resource update) during the next state refresh.
+
+For example, given this previous configuration:
+
+```hcl
+resource "aws_autoscaling_attachment" "example" {
+  autoscaling_group_name = aws_autoscaling_group.example.id
+  elb                    = aws_elb.example.id
+}
+
+resource "aws_autoscaling_group" "example"{
+  # ... other configuration ...
+  load_balancers = [aws_elb.example.id]  
```

Comment:
I think more commonly folks wouldn't have this declared twice (relying on its `Computed: true`) 👍 

```suggestion
```